hr_course/data/ir_cron_data.xml

```diff
@@ -0,0 +1,16 @@
+<odoo>
+    <record id="ir_cron_post_wip" model="ir.cron">
+        <field name="name">HR Course: Check validity date</field>
+        <field name="interval_number">1</field>
+        <field name="interval_type">days</field>
+        <field name="numbercall">-1</field>
```

Comment:
```suggestion
        <field name="numbercall">-1</field>
        <field name="active">False</field>
```
as we going to OCA better to create it in deactivated state as most of users don't need these feature